### PR TITLE
change(batch-requests): limit the body size to 1M

### PR DESCRIPTION
**Pull request by spacewander**
To avoid blocking on reading large body file and
spending too much memory in encode JSON body.

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible?

### CODE REVIEW

File: apisix/core/request.lua [hunk 1]
```diff
@@ -16,12 +16,15 @@
 --
 
 local lfs = require("lfs")
+local log = require "apisix.core.log"
```
Comment (membphis) [inline apisix/core/request.lua:19]: code style: `require("****")`

**Review by membphis — APPROVED** [review]
LGTM

**Comment by membphis** [comment]
CI failed: https://github.com/apache/apisix/pull/2507/checks?check_run_id=1298927664#step:9:138

```
#   Failed test 'TEST 57: route size more than 1.5 MiB - status code ok'
#   at /home/runner/work/apisix/apisix/test-nginx/lib/Test/Nginx/Socket.pm line 935.
#          got: '500'
#     expected: '400'

#   Failed test 'TEST 57: route size more than 1.5 MiB - response_body - response is expected (repeated req 0, req 0)'
#   at /home/runner/work/apisix/apisix/test-nginx/lib/Test/Nginx/Socket.pm line 1589.
#          got: '<html>
# <head><title>500 Internal Server Error</title></head>
# <body>
# <center><h1>500 Internal Server Error</h1></center>
# <hr><center>openresty/1.17.8.2</center>
# </body>
# </html>
# '
#     expected: '{"error_msg":"invalid request body: request size 1678025 is greater than the maximum size 1572864 allowed"}
# '

t/admin/routes.t .......................... 
Dubious, test returned 3 (wstat 768, 0x300)

File: apisix/core/request.lua (hunk 1 above)
Comment (spacewander) [reply apisix/core/request.lua]: @membphis 
Updated

**Review by juzhiyuan — APPROVED** [review]
LGTM

**Pull request merged**